guides/source/upgrading_ruby_on_rails.md

```diff
@@ -277,26 +361,33 @@ I18n.t("missing.key") # didn't raise in 7.0, doesn't raise in 7.1
 I18n.t("missing.key") # didn't raise in 7.0, doesn't raise in 7.1
 ```
 
-Alternatively, you can customise the `I18n.exception_handler`.
-See the [i18n guide](https://guides.rubyonrails.org/v7.1/i18n.html#using-different-exception-handlers) for more information.
+Alternatively, you can customise the `I18n.exception_handler`. See the [i18n
+guide](https://guides.rubyonrails.org/v7.1/i18n.html#using-different-exception-handlers)
+for more information.
 
-`AbstractController::Translation.raise_on_missing_translations` has been removed. This was a private API, if you were
-relying on it you should migrate to `config.i18n.raise_on_missing_translations` or to a custom exception handler.
+`AbstractController::Translation.raise_on_missing_translations` has been
+removed. This was a private API, if you were relying on it you should migrate to
+`config.i18n.raise_on_missing_translations` or to a custom exception handler.
 
 ### `bin/rails test` now runs `test:prepare` task
 
-When running tests via `bin/rails test`, the `rake test:prepare` task will run before tests run. If you've enhanced
-the `test:prepare` task, your enhancements will run before your tests. `tailwindcss-rails`, `jsbundling-rails`, and `cssbundling-rails`
-enhance this task, as do other third party gems.
+When running tests via `bin/rails test`, the `rake test:prepare` task will run
+before tests run. If you've enhanced the `test:prepare` task, your enhancements
+will run before your tests. `tailwindcss-rails`, `jsbundling-rails`, and
+`cssbundling-rails` enhance this task, as do other third party gems.
 
-See the [Testing Rails Applications](https://guides.rubyonrails.org/testing.html#running-tests-in-continuous-integration-ci) guide for more information.
+See the [Testing Rails
+Applications](https://guides.rubyonrails.org/testing.html#running-tests-in-continuous-integration-ci)
+guide for more information.
 
-If you run a single file's tests (`bin/rails test test/models/user_test.rb`), `test:prepare` will not run before it.
+If you run a single file's tests (`bin/rails test test/models/user_test.rb`),
+`test:prepare` will not run before it.
```

Comment:
```suggestion
When running tests for a single file (for example, `bin/rails test test/models/user_test.rb`), the rake `test:prepare` task will not be run.
```

Reply:
will not run before it, makes it sound like it still could be run after or at a different time?